### PR TITLE
Merge of the docker and code repos for AQCU UI. Added travis building. …

### PR DESCRIPTION
**Pull request by zmoore-usgs**
… Added a simple readme.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+  sudo: required
+
+  env:
+    global:
+      - DOCKER_COMPOSE_VERSION=1.22.0
+      - DOCKER_IMAGE=aqcu-gateway
```
Comment (mhines-usgs) [inline .travis.yml]: 👀

File: docker/certificates/import_certs/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+In this folder you should place any certificates that this serivce will need to trust. For the Gateway service that includes certificates for:
```
Comment (mhines-usgs) [inline docker/certificates/import_certs/README.md]: 🍝 i did the same thing, maybe it was a bad idea to be specific in these readmes 😆

File: .travis.yml (hunk 1 above)
Comment (zmoore-usgs) [reply .travis.yml]: I miss this one every time lol

File: docker/certificates/import_certs/README.md (hunk 2 above)
Comment (zmoore-usgs) [reply docker/certificates/import_certs/README.md]: Haha it certainly is not super conducive to mass copy-pasting 😄

**Comment by mhines-usgs** [comment]
Didn't test it out, but if you did, this looks good to me.

**Pull request merged**